### PR TITLE
Improve logging

### PR DESCRIPTION
**Pull request by GJL**
- Upgrade rxjava dependency to 1.3.0
- Sleep after closing coordinators to mitigate race condition

### CODE REVIEW

File: paradox-nakadi-consumer-boot/src/main/java/de/zalando/paradox/nakadi/consumer/boot/components/EventReceiverRegistry.java [hunk 1]
```diff
@@ -214,24 +215,24 @@ public void stop() throws InterruptedException {
             }
         });
 
-        receiverMap.values().forEach(receiver -> {
+        // wait for ZK background tasks
+        Thread.sleep(2000);
```
Comment (lalib) [inline paradox-nakadi-consumer-boot/src/main/java/de/zalando/paradox/nakadi/consumer/boot/components/EventReceiverRegistry.java]: what if it takes 3 or more seconds?

Comment (GJL) [inline paradox-nakadi-consumer-boot/src/main/java/de/zalando/paradox/nakadi/consumer/boot/components/EventReceiverRegistry.java]: The code before my changes was waiting 2000 ms as well. I only moved the line up. This is a hotfix and I don't see an easy way of getting rid of the Thread.sleep.

**Comment by lalib** [comment]
👍

**Comment by GJL** [comment]
👍

**Pull request merged**